pkg/config/azure.go

```diff
@@ -0,0 +1,9 @@
+package config
+
+// AzureConfig specifies the properties required to use Azure as the storage backend
+type AzureConfig struct {
```

Comment:
is this part of this PR?

Reply:
@marpio Overall, the PR looks good to me. Can you answer Marwan's question?